src/test/java/com/google/api/codegen/testdata/discoveries/ruby/ruby_storage.v1.json.baseline

```diff
@@ -1045,6 +1045,11 @@ request_body = Google::Apis::StorageV1::ComposeRequest.new
 # TODO: Assign values to desired members of the `request_body` object.
 
 
+# TODO: Specify destination to download media content. For more information, see:
```

Comment:
It involves adding a parameter to the method signature; without that, it seems misleading to provide a variable declaration.